lib/pry/prompt.rb

```diff
@@ -112,15 +138,15 @@ def prompt_name(name)
     end
 
     add(
-      'simple',
+      :simple,
```

Comment:
why do you insist on using a Symbol for this argument, it is actually just converted back to a string when being stored in a Hash. is it an appearance thing? because one could argue all the other arguments are strings, so this one should be too so we're uniform :P

Reply:
i don't know if that's a ruby convention, maybe a Rails one. np anyway, not an issue here.